enterprise/api/v1/urls.py

```diff
@@ -44,7 +49,7 @@
     ),
     url(
         r'^tableau_token$',
-        views.TableauAuthViewSet.as_view(),
+        views.TableauAuthView.as_view(),
```

Comment:
Why did this change?

Reply:
Ah, because `TableauAuthVew` is a `View` and not a `Viewset` and it Tricked me into believing that I could call `as_view()` on a `ViewSet`